src/sgame/botlib/bot_nav_edit.cpp

```diff
@@ -427,6 +427,132 @@ class AddconCmd: public Cmd::StaticCmd
 	}
 };
 
+static int findClosestNavcon( OffMeshConnections &cons, rVec &targetPoint )
+{
+	int resultIndex = -1;
+	float resultDistance = std::numeric_limits<float>::max();
+	for ( int i = 0; i < cons.offMeshConCount; i++ )
+	{
+		int n = i * 6;
+		// look at start and end points
+		for ( int count = 0; count < 2; count++ )
+		{
+			float distanceX = targetPoint[ 0 ] - cons.verts[ n + count * 3 ];
+			float distanceZ = targetPoint[ 1 ] - cons.verts[ n + count * 3 + 1 ];
+			float distanceY = targetPoint[ 2 ] - cons.verts[ n + count * 3 + 2 ];
+			float distance = sqrt( Square( distanceX ) + Square( distanceY ) + Square( distanceZ ) );
```

Comment:
You should be able to use `Distance`/`DistanceSquared` on rVec (it has an `operator float *`)

Reply:
yes